_i18n/en/resources/moneropedia/mnemonicseed.md

```diff
@@ -7,7 +7,7 @@ summary: "a 13 or 25 word phrase used to backup a Monero account, available in a
 
 ### The Basics
 
-A 13 or 25 word phrase used to backup a Monero account, available in a number of languages. This 25-word phrase (13 words in the case of MyMonero) has all the information needed to view and spend funds from a Monero @account.
+A 16 or 25 word phrase used to backup a Monero account, available in a number of languages. This 25-word phrase (16 words in the case of Feather wallet) has all the information needed to view and spend funds from a Monero @account.
```

Comment:
```suggestion
A 16 or 25 word phrase used to backup and restore a Monero wallet, available in a number of languages. This has all the information needed to view and spend funds from a Monero @account.
```

the mymonero seed was specific to mymonero, but 16 word support is available in most wallets outside of core

maybe add a note about restore height ?